tool/src/main/java/org/wildfly/security/tool/ElytronToolMessages.java

```diff
@@ -332,6 +376,21 @@
     @Message(id = 28, value = "Location parameter is not specified for filebased keystore type '%s'")
     MissingArgumentException filebasedKeystoreLocationMissing(String type);
 
+    @Message(id = 29, value = "Key Pair Algorithm: '%s' is not supported.")
```

Comment:
Are these ids intended? Noticed the other messages in this tool file use "NONE".

Reply:
This file uses IDs for error messages and "NONE" for descriptive messages